crypto/context_local.h

```diff
@@ -0,0 +1,44 @@
+/*
```

Comment:
Please rename this to `include/crypto/context.h` and modify the inclusion lines to `#include "crypto/context.h"`.  Compiler command lines have the necessary flags for this.